cime_config/cesm/allactive/config_compsets.xml

```diff
@@ -40,6 +40,11 @@
 
 
   <compset>
+    <alias>B1D</alias>
```

Comment:
I've tried to introduce some consistency with alias names and although it's a bit cryptic it is consistent.
Since this compset uses non-default versions of cam and clm and the river model I think that the alias name should be BC4L40P1DR